nav2_map_server/src/occ_grid_loader.cpp

```diff
@@ -126,24 +127,31 @@ OccGridLoader::LoadParameters OccGridLoader::load_map_yaml(const std::string & y
   return loadParameters;
 }
 
-nav2_util::CallbackReturn OccGridLoader::on_configure(const rclcpp_lifecycle::State & /*state*/)
+bool OccGridLoader::loadMapFromYaml(
+  std::string yaml_file,
+  std::shared_ptr<nav2_msgs::srv::LoadMap::Response> response)
 {
-  RCLCPP_INFO(node_->get_logger(), "OccGridLoader: Configuring");
-
-  msg_ = std::make_unique<nav_msgs::msg::OccupancyGrid>();
+  if (yaml_file == "") {
```

Comment:
string comparisons should be `yaml_file.empty()`. Small comment, I don't actually care if you change it (I'm certainly guilty of it)

Reply:
Fixed in https://github.com/ros-planning/navigation2/pull/1303/commits/aad70fffc1ca75b888647b077986dbae3a09a7b4